app/uk/gov/hmrc/customsservicestatus/services/test/TestService.scala

```diff
@@ -25,12 +25,14 @@ import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class TestService @Inject() (
-  customsServiceStatusRepository: CustomsServiceStatusRepository
+  customsServiceStatusRepository:      CustomsServiceStatusRepository,
+  adminCustomsServiceStatusRepository: AdminCustomsServiceStatusRepository
 )(implicit ec: ExecutionContext) {
 
   def clearAllData: Future[Unit] =
     for {
       _ <- Mdc.preservingMdc(customsServiceStatusRepository.collection.drop().toFuture())
+      _ <- Mdc.preservingMdc(adminCustomsServiceStatusRepository.collection.drop().toFuture())
```

Comment:
Do we need a call to ensureIndexes below?

Reply:
I meant don't we want to call `ensureIndexes()` on the admin repo here so that old indexes are cleared out when the collection is dropped (just as it's called for the other repo). There are two indexes defined on the admin repo, by the looks.